extra-web/i2pd/autobuild/defines

```diff
@@ -4,9 +4,7 @@ PKGDEP="boost miniupnpc openssl"
 PKGDES="End-to-End encrypted and anonymous Internet"
 
 ABSHADOW=0
-CMAKE_AFTER="-DWITH_AESNI=OFF \
-             -DWITH_AVX=OFF \
```

Comment:
We default to SSE2 target on AMD64, which is why we have disabled AES-NI and AVX.

Reply:
Please let me know if I'm incorrect.

Reply:
At 2.35.0 I've reworked part of AVX and AESNI detection, so it must work without problems.
See our tracker: https://github.com/PurpleI2P/i2pd/issues/1600 and https://github.com/PurpleI2P/i2pd/issues/1602

Reply:
Understood.